skimage/transform/__init__.py

```diff
@@ -1,3 +1,36 @@
+"""This module includes tools to transform images and volumetric data.
+
+- Geometric transformations:
+  These transforms change the shape or position of an image.
+  They are useful for tasks such as image registration,
+  alignment, and geometric correction.
+  Examples: :class:`~skimage.transform.AffineTransform`,
+  :class:`~skimage.transform.ProjectiveTransform`,
+  :class:`~skimage.transform.EuclideanTransform`.
+
+- Image resizing and rescaling:
+  These transforms change the size or resolution of an image.
+  They are useful for tasks such as down-sampling an image to
+  reduce its size or up-sampling an image to increase its resolution.
+  Examples: :func:`~skimage.transform.resize`,
+  :func:`~skimage.transform.rescale`.
+
+- Feature detection and extraction:
+  These transforms identify and extract specific features or
+  patterns in an image. They are useful for tasks s object detection,
+  image segmentation, and  feature matching.
+  Examples: :func:`~skimage.transform.hough_circle`,
+  :func:`~skimage.transform.pyramid_expand`,
+  :func:`~skimage.transform.radon`.
+
+- Image transformation:
+  These transforms change the appearance of an image without changing its
+  content. They are useful for tasks such a creating image mosaics,
+  applying artistic effects, and visualizing image data.
+  Examples: :func:`~skimage.transform.warp`, inverse transforms.
```

Comment:
Usually (see all the rest above), what's under 'Examples' are specific (linked to) functions or classes... Why are 'inverse transforms' in the wild like this? :wink:

Reply:
Probably because there are multiple `iradon*` function's to link to and it wasn't clear which one. I would probably just link [`iradon`](https://scikit-image.org/docs/dev/api/skimage.transform.html#skimage.transform.iradon) as a well-known example.